azkaban-web-server/src/main/java/azkaban/webapp/servlet/PageUtils.java

```diff
@@ -0,0 +1,25 @@
+package azkaban.webapp.servlet;
+
+import azkaban.server.session.Session;
+import azkaban.user.Permission;
+import azkaban.user.User;
+import azkaban.user.UserManager;
+import azkaban.user.UserUtils;
+
+
+public final class PageUtils {
+
+  private PageUtils() {
+
+  }
+
+  public static void hideUploadButtonWhenNeeded(final Page page, final Session session, final UserManager userManager,
+      final Boolean lockdownUploadProjects) {
+    final User user = session.getUser();
+
+    if (lockdownUploadProjects || !UserUtils.hasPermissionforAction(userManager, user,
```

Comment:
I have made changes to make it behave like lockdown.create.projects
i.e.
1. By default everyone can upload
2. When 'lockdown.upload.projects=true', only admin users and users with uploadprojects permission can upload.